.github/workflows/cupertino.yml

```diff
@@ -0,0 +1,59 @@
+name: package:cupertinohttp CI
```

Comment:
I think that my use case might be non-standard because I:
1. need to use flutter to install dependencies
2. have an (explicit) build step
3. use Dart (not flutter) for tests

Reply:
@kevmoo Is this setup possible?